staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -45,7 +47,8 @@ import (
 
 const (
 	// Name of admission plug-in
-	PluginName = "GenericAdmissionWebhook"
+	PluginName       = "GenericAdmissionWebhook"
+	defaultCacheSize = 10
```

Comment:
it can be bigger than 10 (a couple hundred is probably fine)... we just didn't want unbounded growth

Reply:
ok